src/markdown-pages/about_Galasa.md

```diff
@@ -3,19 +3,41 @@ path: "/about"
 title: "About Galasa"
 ---
 
-Galasa is different from other test tools. It is a deep integration test framework, enabling you to test applications that interact with z/OS (including CICS applications) without stubbing or mocking. 
+Galasa is different from other test tools; enabling deep integration testing across platforms and technologies within a DevOps pipeline, Galasa supports repeatable, reliable, agile testing at scale across your enterprise.
 
-Galasa tests are not limited to testing mainframe-specific applications - they can extend to encompass entire families of hybrid cloud applications. Most manual tests can be automated with Galasa, helping you to deliver changes to your core applications quickly and successfully. 
+### Platform integration
+The Galasa framework is built with a native knowledge of z/OS and hybrid multi-cloud, enabling end-to-end testing of applications that run on different platforms without stubbing or mocking. 

```

Comment:
"native knowledge of z/OS and cloud native platforms"

Reply:
have updated